File: 백준/Bronze/1924. 2007년/2007년.c
#include <stdio.h>

int main(void) { 
   int a, b;
   scanf("%d %d", &a, &b);
   
   int month[12] = {31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};

   int date = 0;
   for(int i = 0; i < a - 1; i++){
      date += month[i];
   }

   date += b;

   if(date % 7 == 1)   printf("MON");
   else if(date % 7 == 2)   printf("TUE");
   else if(date % 7 == 3)   printf("WED");
   else if(date % 7 == 4)   printf("THU");
   else if(date % 7 == 5)   printf("FRI");
   else if(date % 7 == 6)   printf("SAT");
   else   printf("SUN");
   
   return 0;
}

File: 백준/Bronze/1924. 2007년/2007년.py
a, b = map(int, input().split())

month = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]

date = 0
for i in range(a - 1):
  date += month[i]

date += b

if date % 7 == 1:
  print("MON")
elif date % 7 == 2:
  print("TUE")
elif date % 7 == 3:
  print("WED")
elif date % 7 == 4:
  print("THU")
elif date % 7 == 5:
  print("FRI")
elif date % 7 == 6:
  print("SAT")
else:
  print("SUN")